tests/unit/acl.tcl

```diff
@@ -261,3 +261,25 @@ start_server {tags {"acl"}} {
         assert_match "*Unknown subcommand or wrong number of arguments*" $e
     }
 }
+
+set server_path [tmpdir "server.acl"]
+exec cp -f tests/assets/user.acl $server_path
+start_server [list overrides [list "dir" $server_path "aclfile" "user.acl"]] {
+    # user alice on allcommands allkeys >alice
+    # user bob on -@all +@set +acl ~set* >bob
+
+    test "Alice: can excute all command" {
+        r AUTH alice alice
+        assert_equal "alice" [r acl whoami]
+        catch {r SET key value} e
+        assert {$e eq "OK"}
```

Comment:
```suggestion
        r SET key value
```
the test will fail anyway if any command errors.

Reply:
updated